src/main/java/nl/tudelft/lifetiles/core/util/Message.java

```diff
@@ -21,6 +21,10 @@
      * A shout message indicating data has been loaded.
      */
     public static final Message LOADED = create("loaded");
+    /**
+     * A shout message indicating annotations have been inserted.
+     */
+    public static final Message ANNOTATIONS = create("annotations");
```

Comment:
This is not a general message and should thus reside in an annotation- or graph-specific class.